src/Route.elm

```diff
@@ -1,11 +1,13 @@
-module Route exposing (Route(..), fromUrl, href, replaceUrl)
+module Route exposing (Route(..), fromUrl, href, replaceUrl, routeToString)
 
+import Basics.Extra exposing (uncurry)
 import Browser.Navigation
 import Html
 import Html.Attributes
 import Url
+import Url.Builder as Builder exposing (QueryParameter)
 import Url.Parser exposing ((<?>))
-import Url.Parser.Query
+import Url.Parser.Query as Query
```

Comment:
I like to avoid using aliases and prefer to use absolute names.

I know it makes this bigger, but I find it much nicer when reading the code. Especially after some time, since I don't have to look at the header where the function is coming from.

If you don't have strong feelings about this I would like to revert this.